src/test/regress/sql/pg13.sql

```diff
@@ -115,5 +115,18 @@ EXPLAIN (ANALYZE TRUE, WAL TRUE, COSTS FALSE, SUMMARY FALSE, BUFFERS FALSE, TIMI
 WITH cte_1 AS (INSERT INTO test_wal VALUES(6,66),(7,77),(8,88) RETURNING *)
 SELECT * FROM cte_1;
 
+CREATE TABLE distributed_table(a int, b int);
```

Comment:
I think we should add a test with https://github.com/citusdata/citus/issues/4129 as well

Reply:
Isn't this test already #4129 ?

Reply:
that's true